packages/next/src/cli/next-start.ts

```diff
@@ -9,28 +9,29 @@ import {
   isPortIsReserved,
 } from '../lib/helpers/get-reserved-port'
 
-type NextStartOptions = {
+export type NextStartOptions = {
   port: number
   hostname?: string
   keepAliveTimeout?: number
 }
 
+/**
+ * Start the Next.js server
+ *
+ * @param options The options for the start command
+ * @param directory The directory to start the server in
+ */
 const nextStart = async (options: NextStartOptions, directory?: string) => {
   const dir = getProjectDir(directory)
-  const host = options.hostname
-  const port = options.port
```

Comment:
These can stay here since they're optimized for change e.g. when we want to validate or sanitize them

Reply:
I'm not sure optimizing for change like in this case is a good philosophy. Assignments like these might add visual clutter and don't convey developer intent imho.

Changed it